### PR TITLE
Initalizing helm chart

### PR DESCRIPTION
**Pull request by kvincent2**

### CODE REVIEW

File: Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+keywords:
+  - "azure active directory"
+  - "okta"
+  - "provisioning"
+  - "SCIM"
+  - "1Password"
+home: https://support.1password.com/scim
+maintainers: # (optional)
+  - name: 1Password Platform Integrations Team
+    email: support+business@1password.com
```
[the remark this answers is not in the document]
Comment (kvincent2) [reply Chart.yaml:14]: I think this is okay. This is the way it is in the DO repo and my IDE verified that the syntax is correct.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,4 @@
 # op-scim-helm
-Helm charts for the op-scim applications
+This repo defines the helm chart for the 1Password SCIM Bridge application.
+
+We intend to use this as a repository that can be referenced in our various kubernetes marketplace solutions. It may also be used outside of that context with an appropriate deployment script.
```
[the remark this answers is not in the document]
Comment (kvincent2) [reply README.md:4]: I'm going to come back to this in the next couple weeks. Thanks for the feedback!

File: op-scim-bridge/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+apiVersion: v2
```
Comment (kvincent2) [inline op-scim-bridge/Chart.yaml:1]: This line has changed. We need to use apiVersion: v2 for helm3. The template files refer to the kubernetes API version, so those are fine at v1.

File: index.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
```
Comment (kvincent2) [inline index.yaml:1]: This file makes it possible to serve up the helm chart package from this repo.

**Comment by kvincent2** [comment]
Please note that the .tgz binary file is a packaged version of our helm chart that is required for using this repo as a source in other apps.

File: index.yaml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    - okta
+    - provisioning
+    - SCIM
+    - 1Password
+    maintainers:
+    - email: support+business@1password.com
+      name: 1Password Platform Integrations Team
+    name: op-scim
+    urls:
+    - op-scim-1.4.3.tgz
```
Comment (grellyd) [inline index.yaml:22]: How was this archive constructed? Just `tar -czvf . ./op-scim-1.4.3.tgz` or something else?

Comment (kvincent2) [reply index.yaml:22]: To produce the tgz, you run `helm package op-scim-bridge`. This will be documented shortly.

Comment (kvincent2) [reply index.yaml:22]: When we release a new version, we will update `/op-scim-bridge/chart.yml` and `/op-scim-bridge/values.yml`. Then run `helm package op-scim-bridge` followed by `helm repo index op-scim-bridge`. We'll need to tag each version with a tag that mirrors the op-scim repo before we attempt to deploy the app. I would not keep old versions in the repo itself, but allow the tags to provide this info.

**Pull request merged**